### PR TITLE
Rum and Coked Out

### PR DESCRIPTION
**Pull request by ChevyChevron**
Adds Rum and Cola which uses the existing Cuba Libre recipe.  Gives the Cuba Libre a new recipe using Rum and Cola mixed with lime juice to better reflect reality.

### CODE REVIEW

**Comment by ChevyChevron** [comment]
!review required

**Comment by ChevyChevron** [comment]
Not sure why that pinged Arrow specifically, but here we are

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/item_worth/reagents.dm [hunk 1]
```diff
@@ -757,4 +757,6 @@
 	value = 500
 
 /datum/reagent/toxin/undead
-	value = 300
\ No newline at end of file
+	value = 300
+/datum/reagent/alcohol/ethanol/rumandcola
```
Comment (Alberyk) [inline code/modules/item_worth/reagents.dm:761]: This should go with the rest of the ethanol datums up there.

**Review by Alberyk — CHANGES_REQUESTED** [review]

File: code/modules/item_worth/reagents.dm (hunk 1 above)
Comment (ChevyChevron) [reply code/modules/item_worth/reagents.dm:761]: I wasn’t sure if I should insert it there or at the end.  I’ll take a look at resolving this when I get home

**Review by Alberyk — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**